src/cascadia/TerminalControl/TermControl.h

```diff
@@ -32,6 +32,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
     struct TermControl : TermControlT<TermControl>
     {
+        using TimeStamp = uint64_t;
+
+        // represents 0.5 seconds (or 5000 ms)
+        const TimeStamp multiClickTimer = 500000;

```

Comment:
Wait, what? so the unit here is.... 100s of microseconds? 

Can you link to the documentation that rationalizes why this is?